packages/aws-cdk-lib/aws-stepfunctions/README.md

```diff
@@ -23,8 +23,8 @@ const submitJob = new tasks.LambdaInvoke(this, 'Submit Job', {
   outputPath: '$.guid',
 });
 
-const waitX = new sfn.Wait(this, 'Wait X Seconds', {
-  time: sfn.WaitTime.secondsPath('$.waitSeconds'),
+const waitX = sfn.Wait.jsonata(this, 'Wait X Seconds', {
+  time: sfn.WaitTime.seconds('{% $states.input.waitSeconds %}'),
```

Comment:
I love that you're providing the new examples on how to use JSONata. But please not directly update the existing documentation. It would be nice to either:
1. have a new section dedicated for JSONata.
2. give both example of using JSONata and JSONPath.

I'll let you choose the options. I know this would be the new preferred way of using SFN, so we can update the README with some description saying that we recmmend using JSONata solution.